.travis.yml

```diff
@@ -4,12 +4,12 @@ branches:
     - master
 language: python
 python:
-  - 2.7
+  - 3.5
```

Comment:
> Are we dropping Python 2 support?

No.

> Why not test both?

We're testing all three of Python 2.7, 3.4, and 3.5, but we're using Tox instead of Travis's matrix functionality.

https://travis-ci.org/AspenWeb/aspen.py/builds/148520550